README.md

```diff
@@ -65,74 +65,42 @@ If you would like to create a new proposal, get started with our
+[component model type system]: https://github.com/WebAssembly/component-model/blob/main/design/mvp/Explainer.md#type-definitions
 
 Note that this is a different sense of "capability" than [Linux
 capabilities](http://man7.org/linux/man-pages/man7/capabilities.7.html)
 or the withdrawn [POSIX
 capabilities](https://archive.org/details/posix_1003.1e-990310), which
 are per-process rather than per-resource.
 
-The simplest representation of handles are values of [reference
-type](https://github.com/WebAssembly/reference-types). References in
-wasm are inherently unforgeable, so they can represent handles directly.
-
-Some programming languages operate primarily within linear memory,
-such as C, C++, and Rust, and there currently is no easy way for
-these languages to use references in normal code. And even if it does
-become possible, it's likely that source code will still require
-annotations to fully opt into references, so it won't always be
-feasible to use. For these languages, references are stored in a
-[table](https://webassembly.github.io/spec/core/bikeshed/index.html#tables%E2%91%A0)
-called a *c-list*. Integer indices into the table then identify
-resources, which can be easily passed around or stored in memory. In
-some contexts, these indices are called *file descriptors* since they're
-similar to what POSIX uses that term for. There are even some tools,
-such as wasm-bindgen, which make this fairly easy. (Internally, tools
-and engines don't always use actual WebAssembly tables to do this,
-however those are implementation details. Conceptually, they work as if
-they had tables.)
-
-Integer indices are themselves forgeable, however a program can only
-access handles within the c-list it has access to, so isolation can still
-be achieved, even between libraries which internally use integer indices,
-by witholding access to each library's c-list to the other libraries.
-Instances can be given access to some c-lists and not others, or even
-no c-lists at all, so it's still possible to establish isolation between
-instances.
-
-Witx-specified APIs use a special `handle` keyword to mark parameters
-and return values which are handles. In the short term, these are
-lowered to integer indices, with an implied table, so that the APIs
-can be easily used from C and similar languages today. Once [interface
-types](https://github.com/WebAssembly/interface-types) is
-ready, we expect to make use of them to provide APIs which can be used
-either from languages using references or from languages using integer
-indices, with tables being used and managed automatically.
-
 ### Interposition
 
-Interposition in the context of WASI interfaces is the ability for a
-Webassembly instance to implement a given WASI interface, and for a
-consumer WebAssembly instance to be able to use this implementation
-transparently. This can be used to adapt or attenuate the functionality
-of a WASI API without changing the code using it.
```

Comment:
I wonder whether it makes sense to keep this explanation in for those who aren't familiar with the concept.

Reply:
Looking at this again, that makes sense. I've now restored that paragraph. Thanks!